.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c  # [sic]. Travis uses objective-c just to determine it needs to use the xcode cli
-osx_image: xcode8.3
+osx_image: xcode9.1
```

Comment:
Why 9.1. Isn't it still beta?

Reply:
Travis [doesn't state anything](https://docs.travis-ci.com/user/reference/osx/#OS-X-Version) the like